rotateList.py

```diff
@@ -0,0 +1,45 @@
+# Definition for singly-linked list.
+# class ListNode(object):
+#     def __init__(self, x):
+#         self.val = x
+#         self.next = None
+
+class Solution(object):
+    def rotateRight(self, head, k):
+        """
+        :type head: ListNode
+        :type k: int
+        :rtype: ListNode
+        """
+        if not head or k == 0:
+            return head
+        count = 0
+        cur = head
```

Comment:
Add comments here to explain what you are doing.
e.g., `# Measuring the length of the linked list.`